presentations/pt-br/index.html

```diff
@@ -0,0 +1 @@
+<meta http-equiv="refresh" content="0; URL=../../index.html" />
```

Comment:
remove this and lift all the files up in the same level as all other presentations